Neos.Flow/Scripts/flow.php

```diff
@@ -55,8 +55,8 @@
         $_SERVER['FLOW_ROOTPATH'] = trim(getenv('FLOW_ROOTPATH'), '"\' ') ?: dirname($_SERVER['PHP_SELF']);
     }
 
-    $context = trim(\Neos\Flow\Core\Bootstrap::getEnvironmentConfigurationSetting('FLOW_CONTEXT'), '"\' ') ?: 'Development';
+    $context = \Neos\Flow\Core\Bootstrap::getEnvironmentConfigurationSetting('FLOW_CONTEXT') ?: 'Development';
 
-    $bootstrap = new \Neos\Flow\Core\Bootstrap($context, $composerAutoloader);
+    $bootstrap = new \Neos\Flow\Core\Bootstrap(trim($context, '"\' '), $composerAutoloader);
```

Comment:
…and no trim here:
```suggestion
    $bootstrap = new \Neos\Flow\Core\Bootstrap($context, $composerAutoloader);
```

Reply:
👍🏻  I didn't expect the elvis operator to support truthy conditions